homeassistant/components/modbus/base_platform.py

```diff
@@ -185,6 +208,16 @@ def unpack_structure_result(self, registers: list[int]) -> str | None:
             v_result = []
             for entry in val:
                 v_temp = self._scale * entry + self._offset
+                if (
+                    self._min_value_threshold is not None
+                    and v_temp < self._min_value_threshold
+                ):
+                    v_temp = self._min_value
+                elif (
+                    self._max_value_threshold is not None
+                    and v_temp > self._max_value_threshold
```

Comment:
Same as min_value, please just use max_value and remove max_value_threshold.

Reply:
min value on its own sets a value it cannot be smaller than BUT ...

This will not work for my use case ... I would have to add another setting

i.e. how do you suggest we achieve: if it is less than 0.025 mA, treat it as 0

I am open to another suggestion, as min_value on its own will not.